pages/index.html

```diff
@@ -24,8 +24,8 @@ <h2>Choose coding, electronics, music&nbsp;or&nbsp;video</h2>
 
   <div class="col col-right">
     <h2>Is there more to life than&nbsp;tech?</h2>
-    <p>Dig into the Bible through short talks, chatting with new friends, and praying together. Ask questions about the Christian faith, and discover what really matters.
-    <p>This year we’ll be taking you into Paul’s letter to the Ephesians, seeing more of God’s plan and purpose for his church.
+    <p>Dig into the Bible through short talks, chatting with new friends, and praying together. Ask questions about the Christian faith, and discover what really matters.</p>
+    <p>In 2020 we'll be exploring the whole story of the Bible, seeing more of God's plan for His people, living in His world, experiencing His blessing.</p>
```

Comment:
curly quotes pls. Also I think you're missing a word?
```suggestion
    <p>In 2020 we’ll be exploring the whole story of the Bible, seeing more of God’s plan for His people, living in His world, and experiencing His blessing.</p>
```